app/src/main/res/layout/fragment_address_editor.xml

```diff
@@ -324,6 +324,41 @@
 
         </com.google.android.material.textfield.TextInputLayout>
 
+        <!-- Country -->
+        <LinearLayout
+            android:id="@+id/country_layout"
+            android:layout_width="match_parent"
+            android:layout_height="wrap_content"
+            android:layout_marginTop="10dp"
+            android:paddingStart="3dp"
```

Comment:
3dp seems pretty specific. is this from a spec or just to mirror the credit card editor layout?

Reply:
This is just mirroring what's in the credit card editor. I believe the plan is to migrate it to spec/compose after MVP is hit.